packages/db-migrations/databases/fxa/patches/patch-138-139.sql

```diff
@@ -0,0 +1,25 @@
+-- Create carts table to store checkout Cart state when creating a subscription.
+-- An FxA user can have more than one Cart record e.g. if they have more than
+-- one subscription where each subscription was created in a different session.
+-- A Cart record may not be associated with an FxA user e.g. if they started
+-- the new user checkout flow but did not get to the point where we create an
+-- account.
+
```

Comment:
The default database and table character set name and collation name is `(utf8mbf, utf8mb4_bin)` in my local MySQL server, so I omitted `SET NAMES utf8mb4 COLLATE utf8mb4_bin;` which was part of a [reference patch](https://github.com/mozilla/fxa/pull/7144) to create the `paypalCustomers` table two years ago. Since then we have e.g. update to MySQL 8.

I asked [in Slack](https://mozilla.slack.com/archives/CLV3KMZ8B/p1688152551684649) if this is the default for the stage/prod MySQL server (I didn't find any config in the cloudops-deployment repo), though I may need to follow up after the long weekend, as @jbuck and @dlactin are both out today.

Reply:
I left it there explicitly as a precaution after discussing in Slack.